containers/detect_rrnas/parse_tRNA.py

```diff
@@ -66,5 +68,9 @@
                         counts = int(line.split(":")[1].split()[0])
                         if (aa_pred in aa or "Met" in aa_pred) and counts > 0:
                             trnas += 1
-
-        print("{name}\t{trnas}".format(name=os.path.basename(args.input).split("_stats")[0], trnas=trnas))
+        if args.output:
+            with open(args.output, "w") as file_out:
+                file_out.write("{name}\t{trnas}\n".format(
```

Comment:
The same here, I would take the new line to the top and the file_our.write it or print it. It's easy to just modify one of them and leave a broken one by mistake